src/internal/operators/ignoreElements.ts

```diff
@@ -28,10 +29,9 @@ class IgnoreElementsOperator<T, R> implements Operator<T, R> {
 /**
  * We need this JSDoc comment for affecting ESDoc.
  * @ignore
- * @extends {Ignored}
  */
 class IgnoreElementsSubscriber<T> extends Subscriber<T> {
-  protected _next(unused: T): void {
-    // Do nothing
+  protected _next(unused: T): Observable<never> {
+    return NEVER;
```

Comment:
it should continue to do nothing. `_next` should never return a value.

Reply:
reverted